sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -319,10 +324,10 @@ def initial_source_split(self, request, unused_instruction_id=None):
     return response
 
   def create_execution_tree(self, descriptor):
-    if descriptor.primitive_transform:
-      return self.create_execution_tree_from_fn_api(descriptor)
-    else:
+    if descriptor.transforms:
       return self.create_execution_tree_from_runner_api(descriptor)
+    else:
```

Comment:
I think you should be able to delete this code as `descriptor.transforms` should always be true or is this because tests still need to migrate?